docs/environments/pendulum.md

```diff
@@ -1,4 +1,56 @@
 # Pendulum
+| Overview          | Specification                                  |
+|-------------------|------------------------------------------------|
+| Observation Space | 3-d array: [cos(theta), sin(theta), theta_dot] |
+| Action Space      | \{0, 1, 2\}                                    |
+| Reward Space      | \{0, 1\}                                       |
+| Loading String    | `'pendulum'`                                   |
+
+
+## Description
 
 Starting from a hanging down position, swing up a single pendulum to the
-inverted position and maintain the pendulum in this position.
\ No newline at end of file
+inverted position and maintain the pendulum in this position.
+
+Most of the default arguments model the pendulum described in "A Comparison of
+Direct and Model-Based Reinforcement Learning" (Atkeson & Santamaria, 1997).
+The key differences are:
+1) This environment is now continuing, i.e. the "trial length" is infinite.
+2) Instead of directly returning the angle of the pendulum in the observation,
+this environment returns the cosine and sine of the angle.
+3) There are only three discrete actions (apply a torque of -1, apply a torque
+of +1, and apply no-op) as opposed to a continuous torque value.
+4) The default reward function is implemented as a sparse reward, i.e. there
+is only a reward of 1 attained when the angle is in the region specified by
```

Comment:
I think it's more direct to say "sparse reward, i.e. when the pendulum is near upright the reward is 1, otherwise 0."

Reply:
Sure, since I'm giving more details in the 'rewards' heading anyway.